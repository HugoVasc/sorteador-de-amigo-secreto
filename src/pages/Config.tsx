import Formulario from "../components/Formulario/Formulario";
import ListaParicipantes from "../components/ListaParticipantes/ListaParticipantes";
import Rodape from "../components/Rodape/Rodape";

const Config = () => {
  return (
    <>
      <Formulario />
      <ListaParicipantes />
      <Rodape />
    </>
  );
};

export default Config;
