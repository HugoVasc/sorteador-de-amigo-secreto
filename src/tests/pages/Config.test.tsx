import { render } from "@testing-library/react";
import React from "react";
import { RecoilRoot } from "recoil";
import Config from "../../pages/Config";

const mockNavegacao = jest.fn();

jest.mock("react-router-dom", () => {
  return {
    useNavigate: () => mockNavegacao,
  };
});

describe("config page", () => {
  test("should render", () => {
    const { container } = render(
      <RecoilRoot>
        <Config />
      </RecoilRoot>
    );
    expect(container).toMatchSnapshot();
  });
});
