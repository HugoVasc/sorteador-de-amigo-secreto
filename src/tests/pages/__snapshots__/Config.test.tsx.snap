// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config page should render 1`] = `
<div>
  <form>
    <input
      class="sc-bczRLJ kTqFYV"
      placeholder="Insira os nomes dos participantes"
      type="text"
      value=""
    />
    <button
      disabled=""
    >
      Adicionar
    </button>
    
  </form>
  <ul />
  <footer>
    <button
      disabled=""
    >
      Iniciar brincadeira
    </button>
  </footer>
</div>
`;
